src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -118,6 +146,12 @@ private void createDataCell(Row<PDPage> row, float width, String catalogObjectDa
         bucketNameCell.setFillColor(java.awt.Color.decode(color));
     }
 
+    private void createKeyValueContentDataCell(Row<PDPage> row, float width, String catalogObjectData, int size) {
```

Comment:
We can rename `size` into `fontSize` and create a constant for it for the two usages of this method.